serverless/src/shared/__tests__/getCsvMetadata.test.js

```diff
@@ -47,7 +54,7 @@ describe('getCsvMetadata', () => {
       const result = await getCsvMetadata('testScheme')
 
       expect(result).toEqual([
-        'Keyword Version: N',
+        'Keyword Version: 20.1',
```

Comment:
This should be pulled out into its own test

Reply:
See refactoring, version check is now in handler.js